pkg/virt-operator/install-strategy/strategy.go

```diff
@@ -235,19 +221,10 @@ func GenerateCurrentInstallStrategy(config *operatorutil.KubeVirtDeploymentConfi
 	}
 	strategy.daemonSets = append(strategy.daemonSets, handler)
 
-	prefix := "system:serviceaccount"
-	typeMeta := metav1.TypeMeta{
-		Kind: customSCCPrivilegedAccountsType,
+	// add all OpenShift specific resources to InstallStrategy
+	if config.OnOpenShift {
```

Comment:
Would it simplify things if we always appended the SCC objects to the strategy.sccs slice, and then only applied them when we're on OpenShift?

Reply:
yes, I check it once in [createOrUpdateSCC](https://github.com/kubevirt/kubevirt/pull/2545/files#diff-d5561d66dec277d1e552dd8ac98e1cfcR1436)